docs/content/en/docs/configuration/command-line.md

```diff
@@ -36,6 +36,7 @@ The following command-line options are supported:
 | [`--verify-hostname`](#verify-hostname)                 | [true\|false]              | `true`                  |       |
 | [`--wait-before-shutdown`](#wait-before-shutdown)       | seconds as integer         | `0`                     | v0.8  |
 | [`--watch-namespace`](#watch-namespace)                 | namespace                  | all namespaces          |       |
+| [`--ignore-ingress-without-class`](#ignore-ingress-without-class)| [true\|false]     | `false`                 |       |
```

Comment:
Use alphabetical order. Add also `v0.10` in the Since column.

Reply:
made the changes, please verify